fs/stat_openbsd.go

```diff
@@ -0,0 +1,28 @@
+// +build openbsd
```

Comment:
Isn't this identical to `stat_linux.go` ? could we use the same file with build tags?

Reply:
oh wow, nice catch!
`stat_linux.go` has to be renamed though. what name should it have?